File: PRAK110_2510817210004_MuhammadRizkyRamadhani.py
import math

alas = 5
tinggi = 12
miring = math.sqrt(alas**2 + tinggi**2)
keliling_segitiga = alas+tinggi+miring
luas_segitiga = 0.5 * alas * tinggi

print(f"""
Diketahui :
Alas = {alas} cm 
Tinggi = {tinggi} cm\n\n
Jawab :
Sisi A = {tinggi} cm 
Sisi B = {miring:.0f} cm 
Sisi C = {alas} cm 
Keliling = {keliling_segitiga:.0f} cm 
Luas = {luas_segitiga:.0f} cm
""")

File: PRAK110_2510817210004_MuhammadRizkyRamadhani.c
#include <stdio.h>
#include <math.h>

int main() {
    int alas = 5;
    int tinggi = 12;
    double miring = sqrt(pow(alas, 2) + pow(tinggi, 2));
    double keliling_segitiga = alas + tinggi + miring;
    double luas_segitiga = 0.5 * alas * tinggi;

    printf("Diketahui :\n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n\n", tinggi);

    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", tinggi);
    printf("Sisi B = %.0f cm\n", miring);
    printf("Sisi C = %d cm\n", alas);
    printf("Keliling = %.0f cm\n", keliling_segitiga);
    printf("Luas = %.0f cm\n", luas_segitiga);

    return 0;
}
